Sources/TuistCore/Automation/XcodeBuildArgument.swift

```diff
@@ -17,6 +17,9 @@ public enum XcodeBuildArgument: Equatable, CustomStringConvertible {
 
     /// To pass additional arguments
     case xcarg(String, String)
+    
+    /// To pass additional flags
+    case xcflag(String)
```

Comment:
Given how we handle the other cases, I would rather add a specific case, instead of a generic `xcflag` case